src/components/metadata/styles/metadata.scss

```diff
@@ -1,8 +1,18 @@
 @import '../../../styles/variables';
-@import '../../../styles/mixins';
+@import '../../../styles/extends';
+
+.kui-theme--light {
+  --color-metadata-bg: #{$color-bg-light-3};
+  --color-metadata-kind-token-bg: #{$color-bg-light-2};
+}
+
+.kui-theme--dark {
+  --color-metadata-bg: #{$color-bg-dark-5};
+  --color-metadata-kind-token-bg: #{$color-bg-dark-6};
+}
 
 .pipeline-metadata {
-  @include sidebarBase;
+  @extend %sidebar;
```

Comment:
The metadata panel font colour is now a dimmer than before, such that it appears different than the font colour of the main sidebar. Seems to be in both themes, more noticeable in the dark theme. I think these should both be the same colour (i.e. brighter font on metadata panel for dark theme, and a lighter font on dark theme)?

Reply:
Good spot. Fixed